src/mudblazor/MudBlazor.Template/Components/Layout/MainLayout.razor

```diff
@@ -33,4 +33,16 @@
     {
         _drawerOpen = !_drawerOpen;
     }
-}
\ No newline at end of file
+}
+
+##else
+@Body
+##endif*@
+@*#if (UseServer || UseWebAssembly) -->
+
+<div id="blazor-error-ui">
```

Comment:
Is there a good MudBlazor version of this?  Like maybe a popover and nice dialog?

Reply:
I noticed that this div comes after the `@code {}` block, can we move it up?

> Is there a good MudBlazor version of the default Blazor error `div`?

Good question, maybe @danielchalmers has something?

Reply:
Good catch, I've added this error `div` to just above the `@code` block